app/src/main/java/org/shadowice/flocke/andotp/Activities/SettingsActivity.java

```diff
@@ -352,10 +352,10 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
                 }
             });
 
-            useAutoBackup = (CheckBoxPreference) findPreference(getString(R.string.settings_key_auto_backup_password_enc));
+            useAutoBackup = (ListPreference)findPreference(getString(R.string.settings_key_auto_backup_password_enc));
             useAutoBackup.setEnabled(BackupHelper.autoBackupType(getActivity()) == Constants.BackupType.ENCRYPTED);
             if(!useAutoBackup.isEnabled())
-                useAutoBackup.setChecked(false);
+                useAutoBackup.setValue(Constants.AutoBackup.OFF.toString());
```

Comment:
Needs ``.toLowerCase(Locale.ENGLISH)`` if we change the resource values to lower case.